.github/ISSUE_TEMPLATE/1-bug_report.yaml

```diff
@@ -0,0 +1,43 @@
+name: Bug report

```

Comment:
@yuandaqing2009 I think it will be better that there is a choice for trial version or open-source version.

Reply:
ok

Reply:
@jianminww added